contracts/diaspore/DebtEngine.sol

```diff
@@ -35,6 +35,21 @@ contract DebtEngine is ERC721Base {
         uint256 _tokens
     );
 
+    event PaidBatch(
```

Comment:
I am not sure if this specific event is required, maybe we can emit the Paid event, if the funcionality of pay and payBatch(1) is exactly the same, this event may be redundant